packages/lesswrong/components/pages/UltraFeedPage.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Components, registerComponent } from '../../lib/vulcan-lib/components';
+import { useStyles, defineStyles } from '../hooks/useStyles';
+import { SECTION_WIDTH } from '../common/SingleColumnSection';
+import { useCurrentUser } from '../common/withUser';
+
+const styles = defineStyles("UltraFeedPage", (theme: ThemeType) => ({
+  root: {
+    display: 'flex',
+    justifyContent: 'center',
+    width: SECTION_WIDTH,
+    marginLeft: "auto",
+    marginRight: "auto",
+    position: "relative",
+    zIndex: theme.zIndexes.singleColumnSection,
+  },
+}));
+
+const UltraFeedPage = () => {
```

Comment:
Visiting this page doesn't enable the feed for the user, right?  (Does anything in this PR allow the user to enable the feed for themselves on the front page?)

Reply:
Yes, it does. If they check the checkbox on this page, it enables it for them on the frontpage. That was the whole motivation